app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,47 @@
+class OrganizationsController < ApplicationController
+  before_filter :authenticate_user!, only: [:create, :new, :edit, :update]
+
+  def index
+    @organizations = Organization.all
+  end
+
+  def show
+    @organization = OrganizationPresenter.new(Organization.find(params[:id]))
+  end
+
+  def new
+    @organization = Organization.new
+  end
+
+  def edit
+    @organization = OrganizationPresenter.new(Organization.find(params[:id]))
+  end
+
+  def update
+    @organization = Organization.find(params[:id])
+    if @organization.update_attributes(org_params)
+      redirect_to @organization
+    else
+      render 'edit'
+    end
+  end
+
+  def create
+    @current_user = current_user
```

Comment:
You shouldn't need to make `@current_user` an instance variable. If tests fail without it, let's figure out what's happening here.